src/core/card/Card.tsx

```diff
@@ -1,49 +1,96 @@
-import { IconArrowRight } from "hds-react";
+import { IconArrowRight, Tag } from "hds-react";
 import React from "react";
 // eslint-disable-next-line import/no-extraneous-dependencies
 import classNames from "classnames";
 
 import styles from "./card.module.scss";
+import Link from "../../common/components/link/Link";
 
 export type CardProps = {
+  ariaLabel?: string;
+  className?: string;
   imageUrl?: string;
+  imageLabel?: string;
+  title?: string;
   subTitle?: string;
   text?: string;
-  title?: string;
+  customContent?: React.ReactNode | string;
+  hasLink?: boolean;
   url?: string;
-  className?: string;
+  withBorder?: boolean;
+  withShadow?: boolean;
+  direction?: "fixed-horisontal" | "fixed-vertical" | "responsive";
+  clampText?: boolean;
 };
 
 export default function Card({
+  ariaLabel,
+  className,
   imageUrl,
+  imageLabel,
+  title,
   subTitle,
   text,
-  title,
+  customContent,
+  hasLink,
   url,
-  className,
+  withBorder,
+  withShadow,
+  direction = "responsive",
+  clampText,
 }: CardProps) {
   return (
-    <div className={classNames(styles.cardWrapper, className)}>
-      {imageUrl && (
-        <div
-          className={styles.imageWrapper}
-          style={{
-            backgroundImage: `url(${imageUrl})`,
-          }}
-        />
-      )}
-      <div className={styles.textWrapper}>
-        {title && <div className={styles.title}>{title}</div>}
-        {subTitle && <div className={styles.subTitle}>{subTitle}</div>}
-        {text && <div className={styles.text}>{text}</div>}
-        {url && (
-          <div className={styles.buttonWrapper}>
-            <a href={url}>
-              <IconArrowRight aria-hidden="true" />
-            </a>
+    <Link href={url} className={className} aria-label={ariaLabel || ""}>
+      <div
+        className={classNames(
+          styles.cardWrapper,
+          withBorder && styles.withBorder,
+          withShadow && styles.withShadow,
+          direction && styles[direction]
+        )}
+      >
+        {imageUrl && (
+          <div
+            className={classNames(
+              styles.imageWrapper,
+              direction && styles[direction]
+            )}
+            style={{
+              backgroundImage: `url(${imageUrl})`,
+            }}
```

Comment:
Like I added in the last pull request also, I think this image should not be set as a background but as an image as it is in the sidebar. https://github.com/City-of-Helsinki/react-helsinki-headless-cms/pull/9#discussion_r869883148. Is there a reason why it shouldn't be with an image? Isn't it more accessible and more describing?

Reply:
https://stackoverflow.com/a/1469139/784642

Reply:
Also, should the card have some kind of default image when there is none given? Here is an example from my collection that I'm doing 
![image](https://user-images.githubusercontent.com/389204/168009511-a6838fbb-2f40-416e-965c-767ca0b6931e.png)